sod/spells.js

```diff
@@ -188,6 +188,33 @@ const spellFunctions = {
   ...priest.spellFunctions,
   18670: handler_bossDropThreatOnHit(0.5), // Broodlord Knock Away
   23339: handler_bossDropThreatOnHit(0.5), // BWL Wing Buffet
+  // Nefarian's warrior class call, force Berserker Stance
+  23397: (ev, fight) => {
```

Comment:
For now I've copied these into sod.js as well. But later we can import era/spells into sod/spells and then we can configure only the differences in sod